PowerEditor/src/WinControls/FileBrowser/fileBrowser.cpp

```diff
@@ -1022,7 +1022,7 @@ void FileBrowser::getDirectoryStructure(const TCHAR *dir, const std::vector<gene
 	::FindClose(hFile);
 }
 
-void FileBrowser::addRootFolder(generic_string rootFolderPath)
+void FileBrowser::addRootFolder(generic_string rootFolderPath, bool shouldExpand)

```

Comment:
I don't see the point of adding this boolean. OTOH, you should using `latestSelectedItem` in `<FileBrowser latestSelectedItem="C:\xxx\yyy\\config.xml">` tag to restore the selection on the last selected file of last session is more meaningful.